<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modifservice</class>
 <widget class="QDialog" name="Modifservice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>751</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>TYPE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Modifier">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(27, 75, 104);
color:white;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>PRIX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IDservice">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>IDSERVICE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PRIXservice">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/icons/Inter.png</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>lavage</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lavage interieur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fast food</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>verifier les pneux</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>soint mecanique</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Modifier un service</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_Modifier</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit_IDservice</zorder>
   <zorder>label_7</zorder>
   <zorder>lineEdit_PRIXservice</zorder>
   <zorder>comboBox2</zorder>
   <zorder>label_8</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
